### DISCUSSION TITLE
Question about tracegraph

### DISCUSSION THREAD

**Discussion by bakerwy** [fumitoh/modelx#90] [Q&A] [answered]
I am working on various functions in lifelib and found the function **tracegraph**.
it doesn't have explanation about the function so.. I wonder how to use them.
Is it a function for visualizing the relations between variables that I coded?

**Comment by fumitoh** [comment, marked as answer]
`Model.tracegraph` is meant for internal use only and is not intended for users. For your information, it returns a graph object that keeps track of the dependencies among calculated cells.

**Reply by bakerwy** [reply]
Thanks for your answer :)
